src/app/[fellowship_id]/bible/plans/page.tsx

```
@@ -0,0 +1,253 @@
+      return <Badge variant="outline">Not Started</Badge>
+    }
+
+    if (stats?.percentage === 100) {
+      return (
+        <Badge variant="default" className="bg-green-500">
+          Completed
+        </Badge>
+      )
+    }
+
+    if (stats?.completedDays ?? 0 > 0) {
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)